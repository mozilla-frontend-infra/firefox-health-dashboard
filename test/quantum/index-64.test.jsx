import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Quantum from '../../src/quantum/index';

it('renders correctly', () => {
  const tree = renderer.create((
    <BrowserRouter>
      <Quantum location={{ pathname: '/quantum/64' }} params={{ match: { bits: 64 } }} />
    </BrowserRouter>
  )).toJSON();
  expect(tree).toMatchSnapshot();
});
