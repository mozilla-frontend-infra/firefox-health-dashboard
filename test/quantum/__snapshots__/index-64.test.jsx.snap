// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="DashboardPage-root-1"
>
  <div
    className="DashboardPage-title-2"
  >
    <h1>
      Quantum
      <small
        className="DashboardPage-subtitle-3"
      >
        Release Criteria Report
      </small>
    </h1>
  </div>
  <div>
    <h2
      className="section-header"
    >
      <span>
         
        Overview
         
      </span>
       
    </h2>
    <div
      className="MuiGrid-container-4 MuiGrid-spacing-xs-24-28"
    >
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics"
      >
        <div
          className="criteria-widget status-null graphic-timeline graphic-widget wide-content"
        >
          <header
            className="sides-padding"
          >
            <div>
              <h3>
                <a
                  href="/quantum/32/bugs"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <span
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "Quantum Flow P1 Bugs Data",
                      }
                    }
                  />
                </a>
              </h3>
              <span
                className="sides-padding"
              />
            </div>
          </header>
          <div
            className="widget-content state-loading"
          >
            <div>
              <div>
                Loading...
              </div>
              <div
                className="legend"
              />
            </div>
            <div
              className="widget-target"
            >
              Target: 
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "Fix P1 Bugs",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics"
      >
        <div
          className="criteria-widget status-unknown widget-countdown narrow-content"
        >
          <header
            className="sides-padding"
          >
            <div>
              <h3>
                <a
                  href="https://wiki.mozilla.org/RapidRelease/Calendar"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <span
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "Release Dates",
                      }
                    }
                  />
                </a>
              </h3>
              <span
                className="sides-padding"
              />
            </div>
          </header>
          <div
            className="widget-content"
          />
        </div>
      </div>
    </div>
  </div>
  <div>
    <h2
      className="section-header"
    >
      <span>
         
        Benchmarks
         
      </span>
       
    </h2>
    <div
      className="MuiGrid-container-4 MuiGrid-spacing-xs-24-28"
    >
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div
          className="criteria-widget status-unknown graphic-widget graphic-timeline"
        >
          <header
            className="sides-padding"
          >
            <div>
              <h3>
                <a
                  href="https://treeherder.mozilla.org/perf.html#/graphs?series=%5Bmozilla-central%2Cf90f8f88d6397a66fb9a586ddf19064f41ee3ce3%2C1%2C1%5D&timerange=7776000"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <span
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "MotionMark HTML suite",
                      }
                    }
                  />
                </a>
              </h3>
              <span
                className="sides-padding"
              >
                <a
                  href="/quantum/windows7-32/motionmark_htmlsuite"
                  onClick={[Function]}
                >
                  Breakdown
                </a>
              </span>
            </div>
          </header>
          <div
            className="widget-content state-loading"
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "Loading Perfherder …",
                }
              }
            />
            <div
              className="widget-target"
            >
              Target: 
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "No regressions",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div
          className="criteria-widget status-unknown graphic-widget graphic-timeline"
        >
          <header
            className="sides-padding"
          >
            <div>
              <h3>
                <a
                  href="https://treeherder.mozilla.org/perf.html#/graphs?series=%5Bmozilla-central%2C4bbe842bb4e492b5f546291db455660287d10d84%2C1%2C1%5D&timerange=7776000"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <span
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "MotionMark Animometer",
                      }
                    }
                  />
                </a>
              </h3>
              <span
                className="sides-padding"
              >
                <a
                  href="/quantum/windows7-32/motionmark_animometer"
                  onClick={[Function]}
                >
                  Breakdown
                </a>
              </span>
            </div>
          </header>
          <div
            className="widget-content state-loading"
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "Loading Perfherder …",
                }
              }
            />
            <div
              className="widget-target"
            >
              Target: 
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "No regressions",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div
          className="criteria-widget status-unknown graphic-widget graphic-timeline"
        >
          <header
            className="sides-padding"
          >
            <div>
              <h3>
                <a
                  href="https://treeherder.mozilla.org/perf.html#/graphs?series=%5Bmozilla-central%2C9585321f110f16414e272956c8892a38d03e7797%2C1%2C1%5D&timerange=7776000"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <span
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "ARES6",
                      }
                    }
                  />
                </a>
              </h3>
              <span
                className="sides-padding"
              >
                <a
                  href="/quantum/windows7-32/ARES6"
                  onClick={[Function]}
                >
                  Breakdown
                </a>
              </span>
            </div>
          </header>
          <div
            className="widget-content state-loading"
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "Loading Perfherder …",
                }
              }
            />
            <div
              className="widget-target"
            >
              Target: 
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "No regressions",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div
          className="criteria-widget status-unknown graphic-widget graphic-timeline"
        >
          <header
            className="sides-padding"
          >
            <div>
              <h3>
                <a
                  href="https://treeherder.mozilla.org/perf.html#/graphs?series=%5Bmozilla-central%2C95535e0278558bd59ae324920ca0469278faadb4%2C1%2C1%5D&timerange=7776000"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <span
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "JetStream",
                      }
                    }
                  />
                </a>
              </h3>
              <span
                className="sides-padding"
              >
                <a
                  href="/quantum/windows7-32/JetStream"
                  onClick={[Function]}
                >
                  Breakdown
                </a>
              </span>
            </div>
          </header>
          <div
            className="widget-content state-loading"
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "Loading Perfherder …",
                }
              }
            />
            <div
              className="widget-target"
            >
              Target: 
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "No regressions",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              Speedometer
            </span>
          </h2>
        </div>
      </div>
    </div>
  </div>
  <div>
    <h2
      className="section-header"
    >
      <span>
         
        Performance Tests
         
      </span>
       
    </h2>
    <div
      className="MuiGrid-container-4 MuiGrid-spacing-xs-24-28"
    >
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              Page load (tp5)
            </span>
          </h2>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              Window Opening (tpaint e10s)
            </span>
          </h2>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              Start-up (sessionrestore)
            </span>
          </h2>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              Start-up (sessionrestore_no_auto_restore)
            </span>
          </h2>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              Start-Up (ts_paint)
            </span>
          </h2>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              Tab Opening (tabpaint)
            </span>
          </h2>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              Tab Animation (TART)
            </span>
          </h2>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              Tab Switch (tps)
            </span>
          </h2>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              SVG (tsvg_static)
            </span>
          </h2>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              SVG (tsvgr_opacity)
            </span>
          </h2>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38"
      >
        <div>
          <h2
            className="PerfherderGraphContainer-title-101"
          >
            <span>
              SVG (tsvgx)
            </span>
          </h2>
        </div>
      </div>
    </div>
  </div>
  <div>
    <h2
      className="section-header"
    >
      <span>
         
        Performance Metrics
         
      </span>
       
    </h2>
    <div
      className="MuiGrid-container-4 MuiGrid-spacing-xs-24-28"
    >
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="winOpen"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Window open
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="tabSwitch"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Tab switch
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="tabClose"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Tab close
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="firstPaint"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                First paint
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="sessionRestoreWindow"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Session Restore Window ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="sessionRestoreStartupInit"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Session Restore Startup Init ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="sessionRestoreStartupOnload"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Session Restore Startup Onload ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="tabSwitchUpdate"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Tab Switch Update ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="gcAnimation"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                GC Animation ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="gpuProcessInit"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                GPU Process Initialization ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="gpuProcessLaunch"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                GPU Process Launch ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="inputEventCoalesced"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Input Event Response Coalesced ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="networkCacheHit"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Network Cache Hit ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="networkCacheMiss"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Network Cache Miss ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="placesAutocomplete"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Places Autocomplete 6  First Results ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="searchServiceInit"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Search Service Init ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="timeToDomComplete"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Time to DOM Complete ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="timeToDomInteractive"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Time to DOM Interactive ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="timeToDomLoading"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Time to DOM Loading ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="timeToFirstInteraction"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Time to First Interaction ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="timeToNonBlankPaint"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Time to Non Blank Paint ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="timeToResponseStart"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Time to Response Start ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="webextBackgroundPageLoad"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Webext Background Page Load ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="webextContentScriptInjection"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Webext Content Script Injection ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="webextExtensionStartup"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Webext Extension Startup ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="timeToLoadEventEnd"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Time to Load Event End ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="timeToDomContentLoadedEnd"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Time to DOM Content Loaded End ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="contentPaintTime"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Content Paint Time ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="pageLoad"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                FX Page Load ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="simpleSessionRestored"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Simple Measures Session Restored ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="scalarFirstPaint"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Scalars Timestamp - First Paint ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-5 MuiGrid-grid-xs-6-38  generic-metrics-graphics photon-perf"
      >
        <div
          className="criteria-widget"
          id="timeToFirstScroll"
        >
          <header>
            <h3
              className="graph-title"
            >
              <a
                className="graph-title-link"
              >
                Time to First Scroll ms
              </a>
            </h3>
          </header>
          <div
            className="graph-subtitle"
          />
          <div
            className="graph"
          >
            <div
              className="graph-legend"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
